romancal/datamodels/schemas/reference_files/referencefile.schema.yaml

```diff
@@ -1,33 +1,35 @@
 %YAML 1.1
 ---
-$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
-id: "http://stsci.edu/schemas/jwst_datamodel/referencefile.schema"
-type: object
-properties:
-  meta:
-    type: object
-    properties:
-      date:
-        title: Date this file was created (UTC)
-        type: string
-      origin:
-        title: Organization responsible for creating file
-        type: string
-      reftype:
-        title: Reference file type
-        type: string
-      pedigree:
-        title: The pedigree of the reference file
-        type: string
-      description:
-        title: Description of the reference file
-        type: string
-      author:
-        title: Author of the reference file
-        type: string
-      history:
-        title: History of file modifications
-        type: string
-      useafter:
-        title: Use after date of the reference file
-        type: string
+$schema: http://stsci.edu/schemas/asdf/asdf-schema-1.0.0
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/referencefile.schema"
+allOf:
+- $ref: ../core.schema
+- type: object
+  properties:
+    meta:
+      type: object
+      properties:
+        date:
+          title: Date this file was created (UTC)
+          type: string
+        origin:
+          title: Organization responsible for creating file
+          type: string
+        reftype:
+          title: Reference file type
+          type: string
+        pedigree:
+          title: The pedigree of the reference file
+          type: string
+        description:
+          title: Description of the reference file
+          type: string
+        author:
+          title: Author of the reference file
+          type: string
+        history:
```

Comment:
I don't know what this attribute is for, but there is a history feature built into ASDF that you might consider using.  DataModel has some special handling around it, see:

https://github.com/spacetelescope/stdatamodels/blob/master/src/stdatamodels/model_base.py#L997

Reply:
Done.